python_modules/libraries/dagster-snowflake/dagster_snowflake/resources.py

```diff
@@ -740,8 +740,8 @@ def fetch_last_updated_timestamps(
         Mapping[str, datetime]: A dictionary of table names to their last updated time in UTC.
     """
     check.invariant(len(tables) > 0, "Must provide at least one table name to query upon.")
-    tables = [table.upper() for table in tables]
-    tables_str = ", ".join([f"'{table_name}'" for table_name in tables])
+    uppercase_tables = [table.upper() for table in tables]
```

Comment:
should fetch_last_updated_timestamps be marked experimental?

Reply:
it should. Will do in another PR.